docs/dev-guides/index.md

```diff
@@ -7,9 +7,9 @@ We're thrilled to have you on board with making the Webaverse a more exciting pl
 
 The Webaverse runs on applications that you can write, package and distribute. These packages are called XRPackages (XRPKs).
 
-We've made it as easy as possible to get started! A few points to start with:
+Follow the guides on the left to get started! A few points to begin with:
 
-- There's no Webaverse SDK! Just make your app using one of the many supported asset types (see below) and quickly package it using our CLI.
+- There's no Webaverse SDK! Instead, make your app using one of the many supported asset types (see below) and package it into an XRPackage (XRPK) using our <a href="https://github.com/webaverse/xrpackage-cli" target="_blank" rel="noopener noreferrer">`xrpk` CLI tool</a> or our simple drag-and-drop UI at <a href="https://xrpackage.org/inspect.html" target="_blank" rel="noopener noreferrer">xrpackage.org</a>.
```

Comment:
I don't think this was addressed and doesn't answer the legitimate question of what the SDK is. When this question has been asked (and it's been asked a lot), users experienced the answer that "there is no SDK" quite unsatisfying and confusing.

I think a better approach is to say that the SDK is found in `xrpk` and to go along with the terms the users use.

Reply:
Oh I see, sorry! Does this look okay?

>The Webaverse SDK can be found in [`xrpk`](https://github.com/webaverse/xrpackage-cli). To make a Webaverse application, you create your app using one of the many supported asset types (see below) and package it into an XRPackage using `xrpk`, or the drag-and-drop UI at xrpackage.org.

Reply:
Yes, I think that is more in line with user's terminology. 👍